pkg/controller/controller.go

```diff
@@ -470,6 +470,17 @@ func getClass(ingressType string) string {
 	return "nginx"
 }
 
+func getIssuerKind(issuerType string) string {
+	switch issuerType {
+	case "ClusterIssuer":
```

Comment:
Please document that `ClusterIssuer` is a valid option at -> https://github.com/openfaas/docs/blob/master/docs/reference/ssl/kubernetes-with-cert-manager.md and in README.md.

Reply:
Done